test/controllers/LandingControllerWithAmendmentsSpec.scala

```diff
@@ -593,39 +535,6 @@ class LandingControllerWithAmendmentsSpec extends AmlsSpec with MockitoSugar wit
                 verify(controller.landingService, never()).refreshCache(any[String](), any(), any())(any[HeaderCarrier], any[ExecutionContext])
               }
             }
-
-            "there are incomplete responsible people" should {
-              "redirect to login events" in new Fixture {
```

Comment:
Should we be replacing these specs with similar however; for redress?

Reply:
Yep, good spot, I hadn't seen that these had been removed